pkg/services/secret_synchronizer.go

```diff
@@ -296,7 +296,7 @@ func (s *SecretSynchronizer) syncRemote(ctx context.Context, pacSecret PacSecret
 		client, ok := s.remoteKubeClients[cluster]
 
 		if !ok {
-			s.logger.Error("Remote client not found: ", cluster)
+			s.logger.Error("[Secret][syncRemote] Remote client not found: ", cluster)
```

Comment:
What are those values between the `cotocos` ([]).

Reply:
Oh, it was to differentiate between the error that came from the Secrets and the ConfigMap synchronizer (original [Secrets Synchronizer](https://github.com/powerhome/keess/blob/890aa522924f2f0430ce9fb86c435b224f8ee335/pkg/services/secret_synchronizer.go#L87) and [ConfigMap Synchronizer](https://github.com/powerhome/keess/blob/890aa522924f2f0430ce9fb86c435b224f8ee335/pkg/services/configmap_synchronizer.go#L87)). Since they both used the same message, it was somewhat harder to debug those error.